src/components/src/VRProductDisplayItem/VRProductDisplayItem.less

```diff
@@ -143,3 +146,40 @@
     color: black;
   }
 
+.outer-html-texture {
+    width: 100%; 
+    height: 100%; 
+    position: fixed; 
+    left: 0; 
+    top: 0; 
+    overflow: hidden;
+    z-index: -2;
+}
+
+.inner-html-texture {
+    // background-image: url(cheesecake.png); 
+    color: white; 
+    width: 500px; 
+    height: 500px; 
+    font-size: 64px; 
+    font-family: monospace; 
+    text-align: center
+}
+
+.html-text-texture {
+    background: rgb(30, 30, 30);
```

Comment:
Should this color be them-able?

Reply:
Yeah it might be better to make it a seperate react component maybe?

Reply:
Made it into its own react component so someone can come in and more easily change it